docs/index.rst

```diff
@@ -3,6 +3,10 @@ JAX: High-Performance Array Computing
 
 JAX is Autograd_ and XLA_, brought together for high-performance numerical computing.
 
+If you're looking to train neural networks, use Flax_.
```

Comment:
```suggestion
If you're looking to train neural networks, use Flax_ and start with its documentation.
```

Sorry to flip back and forth. After reading I think it's nice to say to start here, because otherwise one might think you have to start with optax/orbax documentation.

Sorry!